library/src/main/java/com/android/volley/toolbox/CacheKey.java

```diff
@@ -0,0 +1,14 @@
+package com.android.volley.toolbox;
+
+public class CacheKey {
```

Comment:
Seems like this name should reflect that this is a cache key to be used with images. It doesn't make sense to have a width and height for other cache keys right?